ui/pages/confirmations/confirm/confirm.tsx

```diff
@@ -15,17 +17,15 @@ import syncConfirmPath from '../hooks/syncConfirmPath';
 import { BlockaidAlert } from '../components/confirm/blockaid-alert';
 ///: END:ONLY_INCLUDE_IF
 import { LedgerInfo } from '../components/confirm/ledger-info';
-import { AlertActionHandlerContext } from '../../../hooks/useAlertActionHandler';
-import useConfirmationAlertActions from '../hooks/useConfirmationAlertActions';
+import setConfirmationAlerts from '../hooks/setConfirmationAlerts';
 
 const Confirm = () => {
   setCurrentConfirmation();
   syncConfirmPath();
-
-  const processAction = useConfirmationAlertActions();
+  setConfirmationAlerts();
 
   return (
-    <AlertActionHandlerContext.Provider value={{ processAction }}>
+    <AlertActionHandlerProvider>
```

Comment:
Even if we have no alert actions yet, we don't want to remove the `processAction` property since it's a fundamental part of the provider and alert system.

We could add the `processAction` property back, but define the empty function here since we have no confirmation alert actions yet but the context and provider still require the `processAction`.

Reply:
This is now added inside `AlertActionHandlerProvider`.

Reply:
I appreciate that, but the issue is we now have a non-functional implementation of the context and provider.

Ideally we can leave the context and provider functional, but simply use the empty function for `processAction` where we use it in the `confirm` template as it's the confirmation alerts that don't yet have any actions.

Reply:
I included this back in a different [PR](https://github.com/MetaMask/metamask-extension/pull/24766).